### DISCUSSION TITLE
Should diesel types be serde_json serializable?

### DISCUSSION THREAD

**Discussion by crlsktr** [diesel-rs/diesel#3279] [Q&A] [answered]
I ran into some problems serializing the type PgMoney in my project because the type is defined outside of my package and this blocks me from implementing the traits Serialize and Deserialize, also prevents from using serde_derive Serialize and Deserialize on models that use diesel types. 
It wasn't until I made a pull request that I noticed that diesel has serde_json listed as a dependency and I'm wondering if it's your aim to make the diesel types also serializable via serde_json?

**Comment by weiznich** [comment, marked as answer]
For future reference: https://github.com/diesel-rs/diesel/pull/3277#issuecomment-1218389892
